### ISSUE TITLE
Livechat message limit

### ISSUE THREAD

**Issue by grumpygordon** [RocketChat/Rocket.Chat.Livechat#258]
Hello, our team is using livechat with the bot, here is one interesting thing that I found out. When you send too many messages, you exceed api calls limit, set in rate limiter panel, error is [error-too-many-requests], seems like this limit is for the livechat call to rocketchat, because of that this limit is shared between ALL chats, and I didn't find any way to limit each chat or visitor with it's own number. If there is indeed no such options, then it seems like a bug. Did I miss some setting or is it the way I just described. Also one more thing about limits, currently I have rasa bot connected via webhook, same api limit applies for the webhook, and the [api-bypass-rate-limit] in permissions doesn't help to that.

**Comment by renatobecker** [comment]
You can disable both rate limiters(REST and DDP), so then everything will work fine.
We have use cases that send tons of messages per minute, you just need to turn the rate limiter off.

Thanks.

**Comment by grumpygordon** [comment]
Yeah, but I want to have a limit for each user separately, so they don't spam.
